### ISSUE TITLE
Not accepting external contributions

### ISSUE THREAD

**Issue by michaelurban** [material-foundation/material-color-utilities#18]
The issues tab of several GitHub repos are full of front-end developers asking for updates on Material 3/You. MDC, Angular, everyone is wondering how/when they can start using Material 3. 

Frankly, I'm tried of seeing devs whine, sometimes for years, about Material support in Angular or X/Y/Z projects instead of contributing to the development of the libraries themselves. 

Lack of HCT support is the only thing keeping me from using Material 3's color system in a web app today. I was surprised to see a TypeScript library with no accompanying Sass library. I was shocked to see that this repo is not open to external contributions. I didn't even know it existed until a Google employee pointed me to it. 

I'm willing to do a line-by-line port of the TypeScript `material-color-utilities` to Sass, this week, if that port has a path to release.

Some questions:

1. Is Sass support planned?
2. If Sass support is planned, is development underway?
3. If Sass support is planned, when is it expected to be released?
4. If Sass support is planned, can the development work be done in public?
5. If no one has started working on Sass support, does a port created by an external developer have a chance of being included in this repo?
6. Why is this repo not referenced anywhere on material.io?

**Comment by jpohhhh** [comment]
TL;DR: On external contributions _in general_, we expect to start taking those in Q2 2022.

For SASS / in the short term, we can workaround that by taking the code in any format you provide it, get it checked into Google's internal repo for 3rd party code, allowing us to mirror it here.

Addressed some of this on https://github.com/material-foundation/material-color-utilities/issues/17.

1. My understanding is there's _interest_ but no commitment yet, which means its ~months away
2. Nothing underway yet
3. I don't know the webdev stack as well as I wish I did, I'd take that "late 2022" for MDC Web release as a rough indication that some sort of CSS story would also need to come with that
4. Generally, yes, both Android and Material prefer to have _everything_ public
5. 100%, little more commentary on #17 - TL;DR: everyone is very supportive (and **very** appreciative) of getting any work you do towards this in ASAP. A _true_ contributions accepted setup has to wait till Q2-ish, the code is public right now but as you note, nothing on Material.io yet. So for right now, whatever way you want to share the code is fine, we'll get that into Google's internal repo for 3rd party code, and get it mirroed to GitHub.
6. TL;DR challenging year, it was unexpected that we'd need to reengineer color from the ground up for Material You. That work was done by myself in the Android org, and once it was done, that needed to be shared and explored with Material. We expect there to be a fuller story linked directly to Material in Q1, putting us on pace to share out more often & open to external contributions in Q2.
